geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreParams.scala

```diff
@@ -0,0 +1,127 @@
+
+  val HBaseCatalogParam =
+    new GeoMesaParam[String](
+      "hbase.catalog",
+      "Catalog table name",
+      optional = false,
+      deprecatedKeys = Seq("bigtable.table.name"),
+      supportsNiFiExpressions = true)
+
+  val ConnectionParam =
+    new GeoMesaParam[Connection](
+      "hbase.connection",
+      "Connection",
+      deprecatedKeys = Seq("connection"))
+
+  val ZookeeperParam =
+    new GeoMesaParam[String](
+      "hbase.zookeepers",
+      "List of HBase Zookeeper ensemble servers, comma-separated. " +
+          "Prefer including a valid 'hbase-site.xml' on the classpath over setting this parameter",
+      supportsNiFiExpressions = true)
+
+  val CoprocessorUrlParam =
+    new GeoMesaParam[Path](
+      "hbase.coprocessor.url",
+      "Coprocessor Url",
+      deprecatedKeys = Seq("coprocessor.url"),
+      supportsNiFiExpressions = true)
+
+  val CoprocessorThreadsParam =
+    new GeoMesaParam[Integer](
+      "hbase.coprocessor.threads",
+      "The number of HBase RPC threads to use per coprocessor query",
+      default = Int.box(16),
+      supportsNiFiExpressions = true)
+
+  val RemoteFilteringParam =
+    new GeoMesaParam[java.lang.Boolean](
+      "hbase.remote.filtering",
+      "Remote filtering",
+      default = true,
+      deprecatedKeys = Seq("remote.filtering"))
+
+  val MaxRangesPerExtendedScanParam =
+    new GeoMesaParam[java.lang.Integer](
+      "hbase.ranges.max-per-extended-scan",
+      "Max ranges per extended scan. Ranges will be grouped into scans based on this setting",
+      default = 100,
+      deprecatedKeys = Seq("max.ranges.per.extended.scan"),
+      supportsNiFiExpressions = true)
+
+  val MaxRangesPerCoprocessorScanParam =
+    new GeoMesaParam[java.lang.Integer](
+      "hbase.ranges.max-per-coprocessor-scan",
+      "Max ranges per coprocessor scan. Ranges will be grouped into scans based on this setting",
+      default = Int.MaxValue,
+      supportsNiFiExpressions = true)
+
+  val EnableSecurityParam =
+    new GeoMesaParam[java.lang.Boolean](
+      "hbase.security.enabled",
+      "Enable HBase Security (Visibilities)",
+      default = false,
+      deprecatedKeys = Seq("security.enabled"))
+
+  val ConfigPathsParam =
+    new GeoMesaParam[String](
+      "hbase.config.paths",
+      "Additional HBase configuration resource files (comma-delimited)",
+      supportsNiFiExpressions = true)
+
+  val ConfigsParam =
+    new GeoMesaParam[String](
+      "hbase.config.xml",
+      "Additional HBase configuration properties, as a standard XML `<configuration>` element",
+      largeText = true,
+      supportsNiFiExpressions = true)
+
+  val ArrowCoprocessorParam =
+    new GeoMesaParam[java.lang.Boolean](
+      "hbase.coprocessor.arrow.enable",
+      "Processes Arrow encoding in HBase region servers as a coprocessor call",
+      default = Boolean.box(false),
+      supportsNiFiExpressions = true
```

Comment:
Ok, so as a question, do we want that these options are available in NiFi?

Reply:
they will get added automatically i believe, based on being in the data store factory param list.